lib/putasset.js

```diff
@@ -30,8 +38,66 @@ module.exports = async (token, {owner, repo, tag, filename}) => {
         size,
         url,
     });
+
+    return [
+        "https://github.com",
+        owner,
+        repo,
+        "releases",
+        "download",
+        tag, 
+        path.basename(filename)
+    ].join("/");
+
```

Comment:
Could you please change `double quotes` to `single quotes`

Reply:
Sure my bad.